Update service-fabric-cluster-security.md

aljo-microsoft opened the pull request:
Updating to articulate the change in the SDK's default certificate load behavior

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @narayanannamalai, has been notified to review your proposed change.

TerryLanfear commented:
@NarayanAnnamalai - Hi, 4c74356b41 let me know that there were PRs that seemed to be stalled.  Can you review this PR for them?

NarayanAnnamalai commented:
@4c74356b41 thanks for catching this. 
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

jomolnar commented:
@NarayanAnnamalai Hi Narayan. Please resolve the conflicts to enable a merge. Usually, authors of the files are able to fix merge conflicts using the process here on the [wiki](https://microsoft.sharepoint.com/teams/azurecontentguidance/wiki/Pages/Resolve a local merge conflict yourself.aspx). Thanks.

4c74356b41 commented:
Just a ping ;)

jimdial commented:
@4c74356b41   Thank you for the contribution. Your edits were incorporated into the article outside of this PR. If you have additional edits for it, please file a new PR.  Thank you.

#please-close

aljo-microsoft committed:
Update service-fabric-cluster-security.md

Updating to articulate the change in the SDK's default certificate load behavior
